.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!-- Thank you for contributing a pull request! Here are a few tips to help you:
```

Comment:
Am I correct in guessing that Markdown comments (HTML comments, I guess?) automatically hides these 
blocks on display? Neat!

It will be interesting to see if this creates more or less problems for contributors in terms of leaving accidental comments. What would you say to commenting each line vs. block commenting, to make it easier to "remove" these without accidentally leaving a stray closing comment tag behind?

Reply:
Yes that are HTML comments.

I don't see the need to remove the comments. Usually only the contributor edits the PR description. Hence all comments (expect for the ticket link one) can stay in my opinion.